api/AudioBufferSourceNode.json

```diff
@@ -467,10 +419,10 @@
               "version_added": "14"
             },
             "safari": {
-              "version_added": "6"
+              "version_added": "6.1"
```

Comment:
These is reflecting when `AudioBufferSourceNode.play` was introduced, in WebKit trunk version 537.12.